src/main/resources/com/google/api/codegen/readme.snip

```diff
@@ -0,0 +1,53 @@
+@snippet readme(metadata, samples, installationLines)
+  {@title(metadata)}
+
+  {@quickStart(metadata, samples, installationLines)}
+
+  {@commonLinkDeclarations(metadata)}
+@end
+
+@private title(metadata)
+  @if metadata.hasMultipleServices
```

Comment:
Can we use a variable to represent either 'Clients' or 'Client' so we don't need to duplicate the text string here?